releasenotes/notes/fix_qerror_to_dict-13a7683ac4adddd4.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixes a bug in :meth:`.QuantumError.to_dict` where N-qubit circuit
+    instructions where the assembled instruction always applied to
+    qubits ``[0, ..., N-1]`` rather than the instruction qubits. This
+    bug was also affected device and fake backend noise models.
```

Comment:
"was also affected" should presumably be either "was also affecting" or "also affected"?